pkg/webhook/v1alpha1/placemanifestwork_validating.go

```diff
@@ -55,6 +55,12 @@ func (r *PlaceManifestWorkWebhook) validateRequest(newPlaceWork *workv1alpha1.Pl
 }
 
 func validatePlaceManifests(placeWork *workv1alpha1.PlaceManifestWork) error {
+	if placeWork.Spec.PlacementRef.Name == "" {
```

Comment:
this should be validated by openapi schema here https://github.com/open-cluster-management-io/api/blob/main/work/v1alpha1/types.go#L81

Reply:
do you mean remove this validation and do it in the api repo ?

Reply:
no I mean you could have a marker like this https://github.com/open-cluster-management-io/api/blob/main/addon/v1alpha1/types_clustermanagementaddon.go#L118-L120, that is sufficient to validate if name is empty.

Reply:
okay, just confirm wt to do

Reply:
done